### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by 3TUSK**
Don't merge this yet until I get approval from @tartaricacid and @ETStarLight. Thanks for your understanding. 

Any other inputs are welcome.

### CODE REVIEW

File: src/main/resources/assets/twilightforest/lang/zh_cn.lang
Comment (TartaricAcid) [inline src/main/resources/assets/twilightforest/lang/zh_cn.lang]: 伪金块

**Comment by Drullkus** [comment]
Excellent, thanks for using the 1.12 branch :)

Will wait for other guy before merge

**Comment by liach** [comment]
Minoshroom is not Minotaur. I suggest calling it `米诺菇`.

File: src/main/resources/assets/twilightforest/lang/zh_cn.lang
Comment (liach) [inline src/main/resources/assets/twilightforest/lang/zh_cn.lang]: here

**Comment by 3TUSK** [comment]
Danm.. Two translations swapped their seats...

**Review by TartaricAcid — APPROVED** [review]

File: src/main/resources/assets/twilightforest/lang/zh_cn.lang
Comment (liach) [inline src/main/resources/assets/twilightforest/lang/zh_cn.lang]: Do you need to fix this, too?

Comment (3TUSK) [reply src/main/resources/assets/twilightforest/lang/zh_cn.lang]: Isn't meef == minotaur + beef?

Comment (Drullkus) [reply src/main/resources/assets/twilightforest/lang/zh_cn.lang]: Yes. Alternatively it could also be man+beef. Up to you!

Comment (liach) [reply src/main/resources/assets/twilightforest/lang/zh_cn.lang]: I thought it'd be minoshroom's meat.

**Review by liach — APPROVED** [review]

File: src/main/resources/assets/twilightforest/lang/zh_cn.lang
Comment (3TUSK) [reply src/main/resources/assets/twilightforest/lang/zh_cn.lang]: Seems like you @liach forget the fact that minoshroom drops stroganoff (also the minotaur axe), which... corresponding to the mushroom stew.

Comment (liach) [reply src/main/resources/assets/twilightforest/lang/zh_cn.lang]: eh, I haven't played tf a lot, in fact.

**Comment by ETStareak** [comment]
Checked.It seems nothing uncorrect.I think it is great.

**Comment by liach** [comment]
"uncorrect" -> "incorrect"
"nothing is incorrect" instead of "nothing uncorrect'

**Comment by 3TUSK** [comment]
> Me fail English? That's unpossible!

@ETStarLight https://github.com/TeamTwilight/twilightforest/pull/143/files 右上角 "Review Changes" -> "Approval"

**Review by ETStareak — APPROVED** [review]

**Comment by 3TUSK** [comment]
@Drullkus Ready to go.

**Comment by Drullkus** [comment]
Thank you for your help everyone! :)

**Comment by 3TUSK** [comment]
uh, one more question: is 1.11 branch no longer supported in favor of 1.12.1 branch?

**Comment by williewillus** [comment]
correct

**Comment by 3TUSK** [comment]
*oh yeah i have reason for not doing translation for 1.11*

**Pull request merged**
